int8 status
bool tag1_detected
bool tag2_detected